### DISCUSSION TITLE
Getting LXQt preferences from command line

### DISCUSSION THREAD

**Discussion by hovancik** [lxqt/lxqt#2664] [Q&A] [answered]
Hi there, is there a way to get LXQt preferences from command line? Similar to `gsettings` for example: `gsettings get org.cinnamon.desktop.notifications display-notifications`

I am working on improving Do not disturb support for my app Stretchly (https://github.com/hovancik/stretchly/pull/1523) and I would like to add support for LXQt. The feature is simply put: "Don't show Stretchly notifications when user set DND on their system.", therefore I need a way to query for a user's preference. 

I have this for XFCE, KDE, Gnome, Cinnamon and would like to add more DEs. 

The closest I can see currently is trying to get value from `.config/lxqt/notifications.conf`
I've tried `lxqt-config` and `lxqt-config-notificationd` but they only have `-v` and `-h` options from what I can see on Lubuntu.

Many thanks!

**Comment by stefonarch** [comment, marked as answer]
Probably grepping `notifications.conf` is the only way.

"DND" usually means drag'n drop.

EDIT: I noticed that this file only updates after a first notification if "Don't disturb" is set from tray icon. If set from settings it updates immediately.

**Reply by hovancik** [reply]
Thanks! I will check for file then.
